### DISCUSSION TITLE
They should be able to add a way to give away a free server for a limited time, for example 24 hours.

### DISCUSSION THREAD

**Discussion by beanshost** [Paymenter/Paymenter#520] [Ideas]
You should add a feature that allows users to activate a free server for a limited time, such as 24 hours. This would let users experience the full service without commitment, increasing user engagement and potential conversions to paid subscriptions. After the 24-hour period, the server would either deactivate automatically or require a subscription to continue. To prevent abuse, there could be a limit of one free server per user and account verification via email or phone. This feature could help attract new users and boost overall growth.

**Comment by Xylo01** [comment]
And have the option to change the time interval and have the option if you would like the client to be able to renew

**Comment by finnjosten** [comment]
I think the word you where looking for is "Demo"

**Comment by Xylo01** [comment]
Not really cause we are currently working on a branch of our company for free servers

**Reply by finnjosten** [reply]
But if you want to give out temporary free servers (as mentioned a 24hr limit) doesnt that mean its a demo or trial server?

Because if its a free lineup then just set the price to 0 and done?

Also my comment was directed to the OP.

**Reply by Xylo01** [reply]
It's more of to make sure the service is even being used, because then they order and abandon the server and the only way for us to check is manually then having to go through the system to check is a bit of a hassle especially when you start reaching hundreds or thousands of customers

**Comment by Jay19911** [comment]
make yourself an extension :)
